### PR TITLE
update key setup script and doc

### PR DESCRIPTION
**Pull request by hirokuni-kitahara**
- update key setup script with argparse
- update a doc related to the script

### CODE REVIEW

File: scripts/ACM/acm-verification-key-setup.sh [hunk 1]
```diff
@@ -1,42 +1,118 @@
...
+        esac
+done
+set -- "${POSITIONAL[@]}" # restore positional parameters
+
+if [[ -z $TARGET_LABEL ]]; then
+  TARGET_LABEL=environment=dev
+fi
+
+if [[ -z $KEY_SECRET_NAME ]]; then
+  KEY_SECRET_NAME=keyring-secret
```
Comment (gajananan) [inline scripts/ACM/acm-verification-key-setup.sh:73]: This must

File: scripts/ACM/acm-verification-key-setup.sh [hunk 2]
```diff
@@ -1,42 +1,118 @@
...
+            echo "Invalid input: ${1}"
+            exit 1
+            shift
+            ;;
+        esac
+done
+set -- "${POSITIONAL[@]}" # restore positional parameters
+
+if [[ -z $TARGET_LABEL ]]; then
+  TARGET_LABEL=environment=dev
```
Comment (gajananan) [inline scripts/ACM/acm-verification-key-setup.sh:69]: This must be like below (change from = to :)
TARGET_LABEL=environment:dev

File: docs/ACM/README_SETUP_KEY_RING_ACM_ENV.md [hunk 3]
```diff
@@ -30,10 +30,10 @@ First connect to an ACM hub cluster and execute the [acm-verification-key-setup.
...
 ```
 curl -s  https://raw.githubusercontent.com/open-cluster-management/integrity-shield/master/scripts/ACM/acm-verification-key-setup.sh | bash -s \
-          integrity-shield-operator-system  \
-          keyring-secret  \
-          /tmp/pubring.gpg \
-          environment:dev  |  kubectl apply -f -
+          -namespace integrity-shield-operator-system  \
+          --secret keyring-secret  \
+          --path /tmp/pubring.gpg \
+          --label environment=dev  |  kubectl apply -f -
```
Comment (gajananan) [inline docs/ACM/README_SETUP_KEY_RING_ACM_ENV.md:36]: please change from kubectl to oc  (we planned to change it earlier)

File: docs/ACM/README_SETUP_KEY_RING_ACM_ENV.md [hunk 4]
```diff
@@ -48,9 +48,9 @@ First connect to a ACM hub cluster where a verification key is already setup and
...
 ```
 curl -s  https://raw.githubusercontent.com/open-cluster-management/integrity-shield/master/scripts/ACM/acm-verification-key-setup.sh | bash -s \
-          integrity-shield-operator-system  \
-          keyring-secret  \
-          /tmp/pubring.gpg \
-          environment:dev  |  kubectl delete -f -
+          --namespace integrity-shield-operator-system  \
+          --secret keyring-secret  \
+          --path /tmp/pubring.gpg \
+          --label environment=dev  |  kubectl delete -f -
```
Comment (gajananan) [inline docs/ACM/README_SETUP_KEY_RING_ACM_ENV.md:54]: please change from kubectl to oc  (we planned to change it earlier)

**Review by gajananan — APPROVED** [review]

**Review by yuji-watanabe-jp — APPROVED** [review]

**Review by yuji-watanabe-jp — APPROVED** [review]

**Pull request merged**